client/app/vps/cloud-database/order/vps-cloud-database-order.html

```diff
@@ -0,0 +1,132 @@
+            <oui-select
+                data-name="orderDatacenter"
+                data-placeholder="{{:: 'common_select' | translate }}"
+                data-model="$ctrl.currentOrder.datacenter"
+                data-items="$ctrl.datacenters"
+                data-match="name"
+                data-on-change="$ctrl.showOrRefreshDurations()"
+                data-required>
+                <span data-ng-bind="$item.name"></span>
+            </oui-select>
+        </div>
+
+        <oui-spinner data-size="m" data-ng-if="$ctrl.loading.durations"></oui-spinner>
+
+        <div data-ng-if="!$ctrl.loading.durations && $ctrl.durations">
+
+            <h3 class="mt-5" data-translate="vps_tab_cloud_database_order_period_title"></h3>
+
+            <fieldset>
+                <legend class="border-0"
+                        data-translate="vps_tab_cloud_database_order_period_subtitle"></legend>
+                <oui-radio-group data-model="$ctrl.currentOrder.duration">
+                    <oui-radio data-ng-repeat="duration in $ctrl.durations"
+                               data-value="duration">
+                        <span data-ng-bind="duration.value | duration:'longDate'"></span>&nbsp;:
+                        <span data-ng-bind-html="duration.details.prices | price:$ctrl.user.ovhSubsidiary"></span>
+                        <oui-spinner data-size="s" data-ng-if="!duration.details.prices"></oui-spinner>
+                    </oui-radio>
+                </oui-radio-group>
+            </fieldset>
+
+            <div data-ng-if="$ctrl.currentOrder.duration">
+                <h3 class="mt-5" data-translate="vps_tab_cloud_database_order_contracts_title"></h3>
+
+                <p data-translate="vps_tab_cloud_database_order_contracts_text"></p>
+
+                <div data-ng-if="$ctrl.currentOrder.duration.details.contracts">
+                    <oui-checkbox data-model="$ctrl.currentOrder.contractsAccepted">
+                        {{:: 'vps_tab_cloud_database_order_contracts_accept' | translate }}
+                    </oui-checkbox>
+                    <ul>
+                        <li data-ng-repeat="contract in $ctrl.currentOrder.duration.details.contracts track by $index">
+                            <a data-ng-href="{{contract.url}}" target="_blank" title="{{contract.name}} ({{:: 'core_new_window' | translate }})">
+                                <span data-ng-bind="contract.name"></span> (PDF)
+                                <span class="fa fa-external-link" aria-hidden="true"></span>
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+        <div class="my-5">
+            <div data-ng-if="$ctrl.purchaseOrder.url">
+                <strong class="text-success"
+                        data-translate="vps_tab_cloud_database_order_purchase_order_created">
+                </strong>
+                <br />
```

Comment:
Otherwise you can apply `d-block` to the previous `<strong>` element and get rid of this `<br />`.